autoload/ale/completion.vim

```diff
@@ -678,10 +722,12 @@ function! ale#completion#Done() abort
     let s:last_done_pos = getpos('.')[1:2]
 endfunction
 
+augroup ALECompletionActions
+    autocmd CompleteDone * call ale#completion#HandleUserData(v:completed_item)
```

Comment:
Put this logic into the existing `ale#completion#Done()` function instead. You can start passing `v:completed_item` to that.

Reply:
I believe that `ale#completion#Done()` will not be called when automatic completion is disabled, and the completion is triggered manually (via a hotkey). That's why I created a new group. Perhaps I can modify the `s:Setup` to look like:

```vim
function! s:Setup(enabled) abort
    augroup ALECompletionGroup
        autocmd!
        if a:enabled
            autocmd TextChangedI * call ale#completion#Queue()
        endif
        if a:enabled || g:ale_completion_tsserver_autoimport
            autocmd CompleteDone * call ale#completion#Done(v:completed_item)
        endif
    augroup END

    if !a:enabled && !g:ale_completion_tsserver_autoimport
        augroup! ALECompletionGroup
    endif
endfunction
```

What do you think?